pkg/private/deb/make_deb.py

```diff
@@ -139,7 +139,7 @@ def MakeDebianControlField(name: str, value: str, is_multiline:bool=False) -> st
           '\\n is not allowed in simple control fields (%s)' % value)
 
   lines = value.split('\n')
-  result = name + ': ' +lines[0].strip() + '\n'
+  result = (name + ': ' +lines[0].strip()).strip() + '\n'
```

Comment:
This seems very subtle.  I asked  how can` ("text" + x.strip()).strip()` be any different from `('text' + x.strip()`?
Is this to protect against the case where  x is '\n' because `value` above is `"\nsome text\n..."`
It seems yes, because we add the leading newline in the calls.
Perhaps this could all be more readable if we did not pass in the leading newline and changed above to be
`lines = [x.strip() for x in values.strip().split('\n')]`.

Reply:
name + ‘: ‘ results in a trailing space of the first line is empty (as it should be for these fields).
On the other hand description in the control file has both the summary line that goes straight after the key and a multiline text after. Maybe we should indeed treat the control description field as a special case as right now it’s the only multiline control field.

Reply:
made explicit handling for two separate multiline scenarios with an enum